src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -499,7 +519,16 @@ def makeTransferRucio(self, wflow, dataIn, dids, dataSize, grouping, copies, nod
             self.logger.info("Creating rule for workflow %s with %d DIDs in container %s, RSEs: %s, grouping: %s",
                              wflow.getName(), len(dids), dataIn['name'], rseExpr, grouping)
             try:
-                res = self.rucio.createReplicationRule(dids, rseExpr, **ruleAttrs)
+                # make decision about current workflow, if it is new request we'll create
+                # new replication rule, otherwise we'll move replication rule
+                if wflow.updatedWorkflow:
+                    rids = self.getRids(wflow.getName())
+                    for rid in rids:
+                        # the moveReplcationRule will raise different exceptions
+                        # if move operation is not normal
+                        self.rucio.moveReplicationRule(rid, rseExpr, self.quotaAccount)
```

Comment:
Don't you need to receive the response back from the server?
It should return a list of rule ids (similar to the `createReplicationRule`, which you will then persist in the database (transfer doc).

Reply:
ok, done